src/site/stages/build/drupal/graphql/facilities-fragments/healthCareLocalFacility.node.graphql.js

```diff
@@ -73,8 +72,7 @@ function queryFilter(isMainLocation) {
   return `
     filter: {conditions: [{field: "status", value: "1", operator: EQUAL, enabled: $onlyPublishedContent}, {field: "field_main_location", value: "${
       isMainLocation ? '1' : '0'
-    }", operator: EQUAL}]}, sort: {field: "field_nickname_for_this_facility", direction: ASC}
-  `;
+    }", operator: EQUAL}]}`;
```

Comment:
Should we sort by `title` instead?

Reply:
I don't think it matters - the sort logic from graphql isn't being applied anywhere, and we usually handle sorting in templates, @mpelzsherman